src/internal/sagaHelpers/index.js

```diff
@@ -12,5 +13,6 @@ Therefore yield ${helperName} will return task descriptor to your saga and execu
 const takeEvery = /*#__PURE__*/ deprecate(takeEveryHelper, /*#__PURE__*/ deprecationWarning('takeEvery'))
 const takeLatest = /*#__PURE__*/ deprecate(takeLatestHelper, /*#__PURE__*/ deprecationWarning('takeLatest'))
 const throttle = /*#__PURE__*/ deprecate(throttleHelper, /*#__PURE__*/ deprecationWarning('throttle'))
+const retry = /*#__PURE__*/ deprecate(retryHelper, /*#__PURE__*/ deprecationWarning('retry'))
```

Comment:
`retry` would be a new thing, we don't need to provide old-style export for it with a deprecation warning